docs/source/adb.rst

```diff
@@ -0,0 +1,131 @@
+.. _adb:
+
+####################
+Android Debug Bridge
+####################
+
+.. important:: Android Debug Bridge is required ONLY for those users pairing an Android mobile device with Cozmo. iOS users require *usbmuxd*; instructions for installing this are given on the install pages for each operating system.
+
+Android Debug Bridge (adb) is necessary in order for the Cozmo SDK to work with Android mobile devices. Please ensure adb is installed before attempting to run the Cozmo SDK with a paired Android device.
+
+-----------------
+macOS Installation
+-----------------
+
+1. Type the following into a Terminal window (requires Homebrew to be installed)::
+
+    brew install android-platform-tools
+
+2. Continue to :ref:`final-install` below to complete installation.
+
+--------------------
+Windows Installation
+--------------------
+
+1. Open your internet browser and go to `the Android developer website. <https://developer.android.com/studio/index.html#Other>`_
+2. Scroll down to *Get just the command line tools*. Download the SDK tools package.
+3. If you downloaded the ``.zip`` file instead of the ``.exe`` file, unzip it into your chosen directory.
+4. Run the installer to start the Android SDK Tools Setup Wizard.
+5. The Setup Wizard will direct you to install the Java Development Kit (JDK) if you do not have it installed.
+6. Complete installation of the Android SDK Tools. Take note of the directory it was installed to (e.g. *C:\Program Files (x86)\Android*).
```

Comment:
replace the `*` with double backticks around the filename so that it's escaped correctly